### ISSUE TITLE
Pressing space skips over the buttons that ask you how well you remembered the card

### ISSUE THREAD

**Issue by ChalkHuman** [kenmueller/memorize.ai-web#24]
Pressing space takes the last clicked option and skips over the card

I don't know if that makes sense, but it makes sense in my head

**Comment by aditya1rawat** [comment]
i think most of the time, when you press space, it flips over the card

**Comment by ChalkHuman** [comment]
I don't think so because I was able to recreate it on multiple devices, even with a hard refresh

**Comment by aditya1rawat** [comment]
Wait. are you saying that pressing space does that action or it should?

**Comment by ChalkHuman** [comment]
It skips the card and it takes whatever the user picked last

It sound confusing, I can explain on a call or something if you want

**Comment by aditya1rawat** [comment]
Ooooh I see. So if I pick struggling on one card, the next card will select struggling if I clicked the spacebar?

**Comment by ChalkHuman** [comment]
Yeah, that's what happens
